dotcom-rendering/src/components/Disclaimer.tsx

```diff
@@ -8,20 +10,56 @@ const disclaimerStyles = css`
 	sup {
 		font-size: 85%;
 	}
-
 	margin-bottom: 16px;
+	color: inherit;
+	text-decoration: none;
+`;
+
+const disclaimerInlineStyles = css`
+	margin-bottom: ${space[1]}px;
+	float: left;
+	clear: left;
+	width: 8.75rem;
+	margin-right: 20px;
+	background-color: ${themePalette('--rich-link-background')};
+	:hover {
+		background-color: ${themePalette('--rich-link-background-hover')};
+	}
+	padding-top: 2px;
+	padding-right: 5px;
+	padding-left: 5px;
+	padding-bottom: 5px;
 `;
 
-type Props = {
-	html: string;
-};
+const DisclaimerText = () => (
+	<p>
+		<sup>
```

Comment:
Why is a `sup` element used here? It doesn't seem to fit with the usage notes or examples in the MDN docs: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/sup#usage_notes

Maybe an `aside` would be a more appropriate element for this block of text?

Reply:
Ah, I see you're already using an aside in the `AffiliateDisclaimer` component. I guess my question would then be: is the `sup` element necessary?

Reply:
This was carried over from Frontend but good shout, I'll remove the sup.